### DISCUSSION TITLE
[Run issue]: Run an older version of TOPAS and OpenTOPAS on the same machine

### DISCUSSION THREAD

**Discussion by alex-phy** [OpenTOPAS/OpenTOPAS#111] [Run-Problem]
### Contact Details

guimaraes.alexf@gmail.com

### What was the issue?

Hi, 

I would like to know if it is possible to keep an older version of TOPAS and install OpenTOPAS on the same machine without causing any conflicts in the simulations.

### TOPAS category

Geometry

### Version

3.7

### What Operative System (OS) are you running on?

Linux

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Attestation

- [X] I testify that I have fully read the bug reporting guideline beforehand.

**Comment by icmeyer** [comment]
The short answer to this is: no there will not be issues. 

The long answer is that it depends on you knowing how your machine is configured and how the two different versions of OpenTOPAS are installed. The base geant4 version is different for different versions of OpenTOPAS. That means if you are building from source it requires maintaining two different geant4 installations and two different OpenTOPAS installations, and then being confident about which one you are calling at any given moment. My opinion is: If your use case is not OpenTOPAS development or comparing different OpenTOPAS versions, it generally is more confusing and frustrating to have two versions of the same code installed and instead have one version that works as expected. If you are using an older version and contemplating a new version, you may want to install the new version, test it out to verify it is working, and then remove the old version.
